docs/personalization/recommendation_models.md

```diff
@@ -244,4 +244,26 @@ The logic used for resolving a submodel is as follows:
     for requesting recommendations. 
     Recommendation are fetched from all the submodels and merged based on the weight (relevance). 
     If one of the submodels delivers recommendations with better relevance, 
-    the results of other models can disappear from the list.
\ No newline at end of file
+    the results of other models can disappear from the list.
+
+### Time-slot based models
+
+Time-slot based models consider only a particular range of time rather than the full day, while creating recommendations.
+They can be used for [popularity](recommendation_models#popularity-models) and [collaborative](recommendation_models#collaborative-models) types of models.
+With time-slot based models, user can configure and set specific time slots.
+
+These time slots:
+
+- can cover any time frame, including minutes, and don't necessarily have to start and end with full hour
+- cannot overlap, for example, you cannot set up 8 a.m - 11 a.m and 9 a.m - 12 a.m slots at once
+- cannot pass between two days, for example, you cannot set slot 11 p.m - 3 a.m
+
+In these models, recommendations are created for both configured time slots and for the main model (for example, for the last 30 days).
+However, time-slot based recommendations are shown as priority in the hours for which time slots are configured (if requested in recommendation call).
+
+To use time slot-based models, this feature must be enabled.
```

Comment:
Time-slot based models are described in separate paragraph. 
I don't see a point to move this information to line 191.

Reply:
It's your call, however, you added Time-slot models under the "## Submodels" header, which calls for at least mentioning them in an introductory section. "Apart form regular submodels that do this and that you can also apply time slot-based models, which modify the results coming by revommendation or popularity models to provide more hits better aligned with the behavior of users at certain times of the day."

What you could do is rename "Submodels" to, for example, "Modify models" and "Nominal attributes" to "Nominal attribute-based sumbodels", "Numeric attributes" to "Numeric attribute-based sumbodels".
This would bring a little order into this section.

Reply:
I'll just change the level of Time-slot based models: ## instead of ###.